lua/luasnip/init.lua

```diff
@@ -74,7 +73,7 @@ end
 ---Retrieve information about snippets available in the current file/at the
 ---current position (in case treesitter-based filetypes are enabled).
 ---@generic T
----@param snip_info (fun(LuaSnip.Snippet): T)? Optionally pass a function that,
+---@param snip_info? (fun(LuaSnip.Snippet): T) Optionally pass a function that,
```

Comment:
Oh that's pretty cool, makes it easier to see which parameters are optional!
It seems like the `?` is still appended to the type (like before), so if we want to have a description like `available(snip_info?): {[string]: T}`, we'd have to do a tiny tweak to `luals-mdgen`. Seems very feasible :)

Reply:
https://github.com/L3MON4D3/luals-mdgen/commit/896453c24b1d2170e546be282d61fd9edf2a5de6